.travis.yml

```diff
@@ -2,4 +2,4 @@ language: node_js
 node_js: 6.9.1 # latest LTS
 cache:
   directories:
-    - phantomjs
+    - node_modules/phantomjs-prebuilt
```

Comment:
Travis invokes `npm install` directly by default for any node projects. `npm install` will only skip install the `phantomjs-prebuilt` package.

https://travis-ci.org/github/time-elements/builds/178111851#L149